assets/js/modules/analytics/common/account-create.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * AccountCreate component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState, Fragment, useCallback, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../../components/button';
+import ProgressBar from '../../../components/progress-bar';
+import { trackEvent } from '../../../util';
+import TimezoneSelect from './timzezone-select';
+import AccountField from './account-field';
+import PropertyField from './property-field';
+import ProfileField from './profile-field';
+import { STORE_NAME } from '../datastore/constants';
+
+import Data from 'googlesitekit-data';
+const { useDispatch, useSelect, select: directSelect } = Data;
+
+const AccountCreate = () => {
+	const { siteName, siteURL } = global.googlesitekit.admin;
+	let tz = directSelect( 'core/site' ).getTimezone();
+	const url = new URL( siteURL );
+	const { createAccount } = useDispatch( STORE_NAME );
+	const accountTicketTermsOfServiceURL = useSelect(
+		( select ) => {
+			return select( STORE_NAME ).getAccountTicketTermsOfServiceURL();
+		},
+		[]
+	);
+	const [ isNavigating, setIsNavigating ] = useState( false );
+	const handleSubmit = useCallback( ( accountName, propertyName, profileName, timezone ) => {
+		trackEvent( 'analytics_setup', 'new_account_setup_clicked' );
+		setIsNavigating( true );
+		async function send() {
+			await createAccount( {
+				accountName,
+				propertyName,
+				profileName,
+				timezone,
+			} );
+
+			// Redirect if the accountTicketTermsOfServiceURL is set.
+			if ( accountTicketTermsOfServiceURL ) {
+				location = accountTicketTermsOfServiceURL;
+			}
+			setIsNavigating( false );
+		}
+		send();
+	} );
+
+	// Fall back to the browser timezone if the WordPress timezone was not set.
+	if ( ! tz || '' === tz || 'UTC' === tz ) {
```

Comment:
I will remove the special handling for UTC. I was under the impression this wasn't valid :) I'll also remove the browser timezone fallback since not the timezone will always be 'set'

Reply:
i kept the fallback for the case where we can't locate a matching timezone in the dropdown (as is the case with UTC, sorry WordPress).